R/fb_league_stats.R

```diff
@@ -32,26 +32,27 @@
   } else {
 
     rlang::inform(
-      'Please be aware that `fb_league_stats(..., team_or_player = "player")` depends on promises, which may not always work.',
+      'Please be aware that `fb_league_stats(..., team_or_player = "player")` depends on `rvest::read_html_live` (and chromote), which may not always work.',
       .frequency = "once",
       .frequency_id = "fb_league_stats-player"
     )
 
-    session <- worldfootballr_chromote_session(url)
-    player_table <- worldfootballr_html_player_table(session)
-    session$session$close(wait_ = FALSE)
+    page <- rvest::read_html_live(url)
+    ## for keepers: although URLs have plural term, div elements have singular term
+    stat_type <- gsub("keepers", "keeper", stat_type)
+    player_table_element <- page$html_elements(paste0("#div_stats_", stat_type))
```

Comment:
looks for this ID element in the HTML comment that is loaded client side

![image](https://github.com/user-attachments/assets/97af5d51-6c49-4ada-8788-90a223e3f973)

Reply:
before, i hardcoded a search for the `#stats_shooting_sh` element. in hindsight, i think this meant that `fb_league_stats(..., team_or_player = "player")` only worked for `stat_type = "shooting"`?! maybe my thought that our prior code was unreliable was actually more so due to a bug than bad code!

Reply:
Hahaha a very nice unintended fix!